ui/main/src/app/business/services/events/opfabEventStream.service.ts

```
@@ -177,4 +177,8 @@ export class OpfabEventStreamService {
     static getMonitoringConfigChangeRequests(): Observable<void> {
         return OpfabEventStreamService.monitoringConfigChange.asObservable();
     }
+
+    static getCurrentPeriod(): {start: number; end: number} {
+        return this.currentPeriod;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)